deploy/operator.yaml

```diff
@@ -29,9 +29,7 @@ spec:
               fieldPath: metadata.name
         - name: WATCH_NAMESPACE
         - name: CONVERSION_CONTAINER
```

Comment:
Can probably remove the environment variable entirely for now. We should probably also drop the `ensure_ims_config` from the hyperconverged controller.